scheduler/src/migrations/20241009102213_add_is_success_data_table.sql

```diff
@@ -0,0 +1,5 @@
+-- Rename bms_data table to worker_data
+ALTER TABLE bms_data RENAME TO worker_data;
+
+-- Add is_success to data table
+ALTER TABLE worker_data ADD COLUMN is_success BOOLEAN DEFAULT FALSE;
```

Comment:
I know we need default for adding the column, but maybe lets drop it after column is added?

Reply:
actually I think that its not required unless there is `NOT NULL` statement